builder/dockerfile/parser/parser_test.go

```diff
@@ -159,3 +160,14 @@ RUN indented \
 	assert.Contains(t, warnings[1], "RUN another     thing")
 	assert.Contains(t, warnings[2], "will become errors in a future release")
 }
+
+func TestParseReturnsScannerErrors(t *testing.T) {
+	label := strings.Repeat("a", 65536)
```

Comment:
For the sake of consistency, I suggest using `bufio.MaxScanTokenSize` instead.

Reply:
F